src/pytest_inmanta_lsm/plugin.py

```diff
@@ -219,9 +218,8 @@ def remote_orchestrator_shared(
     Shared project to be used by the remote orchestrator. Ensures the module being tested is synced to the remote orchestrator
     even if it is a v2 module.
     """
-    in_place = inm_mod_in_place.resolve(request.config)
-    # no need to do anything if this version of inmanta does not support v2 modules or if in_place already adds it to the path
-    if hasattr(module, "ModuleV2") and not in_place:
+    # no need to do anything if this version of inmanta does not support v2 modules
+    if hasattr(module, "ModuleV2"):
```

Comment:
Perhaps it would be useful for readability purposes to extract this block into a `verify_v2_install` method or something? I probably should have done this in the first place back when I implemented this. Not a strict change request.

Reply:
Good idea, done